gateway-ha/pom.xml

```diff
@@ -262,6 +262,18 @@
             <version>${dep.activejdbc.version}</version>
```

Comment:
Remove this and any traces of it?

Reply:
I don't understand your suggestion? We can't remove the line or the dependency as it's still used.

Reply:
I misdunerstood the scope of the PR .. i just glanced and thought this removes all usage of activejdbc .. but that is not the case.. not yet ;-)